io/io/inc/TFileMergeServer.h

```diff
@@ -0,0 +1,85 @@
+#ifndef ROOT_TFileMergeServer
```

Comment:
Sure, I will add them prior to merging. I just don't know how to assign authorship. Most of the code was written by someone else (i.e. Witek or Philippe).